src/battle/common/actor/duplighost/ghost_bombette.inc.c

```diff
@@ -18,7 +18,7 @@ s32 N(DefaultAnims)[] = {
     STATUS_KEY_STATIC,    ANIM_BattleBombette_Idle,
     STATUS_KEY_PARALYZE,  ANIM_BattleBombette_Still,
     STATUS_KEY_DIZZY,     ANIM_BattleBombette_Injured,
-    STATUS_KEY_FEAR,      ANIM_BattleBombette_Injured,
+    STATUS_KEY_UNIMPLEMENTED, ANIM_BattleBombette_Injured,
```

Comment:
Unfortunate that this is too long. No big deal, will be cleaned with clang-format in future should we finally decide to do it

Reply:
In pmret its actually STATUS_KEY_UNUSED, which I like better as a name, but that conflicts with the UNUSED macro in DX in the damage calc functions where the macro INFLICT_STATUS is defined, the interaction of the two macros INFLICT_STATUS(UNUSED) becomes an unintended error. The plan here is to rewrite the status effect system, in which case the INFLICT_STATUS macros won't be needed anymore and this status key can use the same name as on pmret.
edit: actually I'll switch to STATUS_KEY_UNUSED now and just not use the INFLICT_STATUS macro for this status effect specifically.